src/webserver.js

```diff
@@ -85,6 +85,7 @@ function initializeNodeBB(callback) {
 		function(next) {
 			plugins.init(app, middleware, next);
 		},
+		async.apply(plugins.fireHook, 'static:assets.prepare', []),
```

Comment:
Right. NodeBB doesn't do anything with the returned data, actually. It's just for plugins to consume. In this case, it can be removed... or rather, replaced with `{}`, since core expects a parameter here.